src/Backend/Form/Type/CheckboxEnableFieldType.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Backend\Form\Type;
+
+use Backend\Form\DataTransferObject\CheckboxEnabledFieldDataTransferObject;
+use Common\Form\SwitchType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CheckboxEnableFieldType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add(
+                'enableField',
+                SwitchType::class,
+                [
+                    'required' => false,
+                    'label' => $options['enable_field_label']
+                ]
+            )
+            ->add(
+                'field',
+                TextType::class,
```

Comment:
I would make the type of field configurable and the options for it as well, that way it is way more reusable a little bit like how you can configure a CollectionType

Reply:
@carakas I would do this in a later stage. This is just the initial start of using more Vue components. With this we can see if this works in real projects and improve it based on the actual feedback.